crates/bevy_tasks/src/task_pool.rs

```diff
@@ -106,14 +106,10 @@ impl TaskPoolBuilder {
 /// will still execute a task, even if it is dropped.
 #[derive(Debug)]
 pub struct TaskPool {
-    /// The executor for the pool
-    ///
-    /// This has to be separate from TaskPoolInner because we have to create an `Arc<Executor>` to
-    /// pass into the worker threads, and we must create the worker threads before we can create
-    /// the `Vec<Task<T>>` contained within `TaskPoolInner`
+    /// The executor for the pool.
     executor: Arc<async_executor::Executor<'static>>,
 
-    /// Inner state of the pool
+    // The inner state of the pool.
     threads: Vec<JoinHandle<()>>,
```

Comment:
`threads` is documented with a normal comment while `executor` is documented with a doc comment. It would be a good idea to pick one type of comment and stick with it. Note that doc comments on private fields don't seem to show up  in the documentation even when `--document-private-items` is used.

Reply:
Note that doc comments on private fields don't seem to show up in the documentation even when `--document-private-items` is used.

Reply:
> You seem to have accidentally changed the `threads` doc comment into a normal comment.

This was intentional. The comment used to refer to a single field, `TaskPoolInner`, but later was split into two separate fields. Instead of duplicating the doc comment, I chose to make it a regular one.